config/crd/bases/http.keda.sh_httpscaledobjects.yaml

```diff
@@ -60,6 +60,26 @@ spec:
           spec:
             description: HTTPScaledObjectSpec defines the desired state of HTTPScaledObject
             properties:
+              fallbackTargetRef:
```

Comment:
[fallback](https://keda.sh/docs/2.14/concepts/scaling-deployments/#fallback) has a specific meaning in KEDA, isolated from KEDA ecosystem the naming makes sense but here I think we should ideally call it something else, few alternatives that came to my mind
* `coldStartTimeoutRef` - we should probably explicitly mention this is for cold-starts only and technically this serves the request after the `waitTimeout` expires
* `coldStartFailoverRef` - failover might have similar meaning as fallback but doesn't carry the existing overloaded usage with KEDA

Reply:
To be more explicit it could also be something like `coldStartTimeoutFailoverRef` ?

Reply:
yeah, I'd be ok with `coldStartTimeoutFailoverRef`, thoughts @zroubalik @JorTurFer?

Reply:
fine by me, let's go ahead with this one